src/index.js

```diff
@@ -16,22 +16,26 @@ function makeUrlAbsolute(url) {
 class WebSdkApi extends EventEmitter {
   constructor(child) {
     super();
-    this.available = () => {}; // Starts as a no-op, overwritten as a resolve once .ready is called
     this.child = child;
     child.msg_bus.on("alert", (event, ...args) => this.emit(event));
     child.msg_bus.on("signal", signal => this.emit('signal', signal));
     child.msg_bus.on("unrecoverable-agent-state", () => this.emit('unrecoverable-agent-state'));
-    child.msg_bus.on("unavailable", () => this.emit('unavailable'));
-    child.msg_bus.on("available", () => {
-      this.available()
+    child.msg_bus.on("unavailable", () => {
```

Comment:
"unavailable" means "You can't make zome calls". It will only ever be emitted if you've successfully loaded chaperone into the iframe.

If you haven't successfully loaded chaperone into the iframe then `WebSdk.connect` will throw a timeout error.

The exception to this is if you provide a chaperone url to `WebSdk.connect` that has some website on it, but it's not chaperone. Then whatever is there will appear in the iframe.

I'm gonna add an issue to this repo and get a ticket put in v1 for this, but as it's beyond feature parity and not part of the scope of this PR I'd still like to merge.

Does that work for you?